downloads.markdown

```diff
@@ -6,156 +6,183 @@ isDownloads: true
-*   Finally, you can ask questions of other Haskell users and experts on the [#haskell IRC channel](irc://irc.freenode.net/haskell) on the Freenode IRC network.
+*   [Using a package manager on linux](#package-manager)
 
-## Stack { #stack }
+*   [Official bindists](#bindists)
 
-### What it is
+*   [Building from source](#from-source)
 
-Stack is a cross-platform build tool for Haskell that handles management of the toolchain (including the GHC compiler and MSYS2 on Windows), building and registering libraries, and more.
+*   [Other options](#other-options)
 
-### What you get
+### Using a package manager on Linux { #package-manager }
 
-*   Once downloaded, it has the capacity to download and install GHC and other core tools.
-*   Project development is isolated within sandboxes, including automatic download of the right version of GHC for a given project.
-*   It manages all Haskell-related dependencies, ensuring reproducible builds.
-*   It fetches from a curated repository of over a thousand packages by default, known to be mutually compatible.
-*   It can optionally use Docker to produce standalone deployments.
+Refer to your distribution package manager documentation. For convenience, below are a few distribution specific instructions, outlining 3rd party repository use as well.
 
-### How to get it
+*Note: Most linux distros don't allow different versions of GHC to be installed in parallel. Sometimes the GHC packages are a bit outdated as well. In either case, consider using [GHCup](#ghcup) instead.*
 
-The [install and upgrade page](http://docs.haskellstack.org/en/stable/install_and_upgrade/) describes how to download Stack on various platforms, although the main three are repeated here:
+*Note: haskell-language-server isn't packaged by most distributions. Instead you will have to install it manually, see the [release page](https://github.com/haskell/haskell-language-server/releases). This is optional.*
 
-*   [Ubuntu Linux](http://docs.haskellstack.org/en/stable/install_and_upgrade/#ubuntu)
-*   [OS X](http://docs.haskellstack.org/en/stable/install_and_upgrade/#os-x)
-*   [Windows](http://docs.haskellstack.org/en/stable/install_and_upgrade/#windows)
+<p><a data-toggle="collapse" href="#collapse-linux" class="btn btn-xs btn-primary">Show linux distros</a></p>
 
-Instructions for other Linux distributions, including Debian, Fedora, Red Hat, Nix OS, and Arch Linux, are also available.
+<div id="collapse-linux" class="collapse">
+##### <span style="text-decoration: underline">Ubuntu</span>
 
-### Where to get help { #stackhelp }
+Steps to setup ghc and cabal are given in the [ghc ppa](https://launchpad.net/~hvr/+archive/ubuntu/ghc)
 
-For help with Haskell and GHC in general, see the links mentioned [above](#help). For Stack itself there are also the following resources:
+Packages from the PPA can be installed as follows:
+```bash
+sudo add-apt-repository -y ppa:hvr/ghc
```

Comment:
Have we managed to emancipate ourselves from hvr now that he's gone missing? Is the PPR updated by someone else?

Reply:
Whether he's missing or not... the PPA provides cabal-3.4 and GHC-9.0.1, so I consider this recent.